x/fees/types/msg.go

```diff
@@ -26,11 +26,16 @@ func NewMsgRegisterFee(
 	withdrawal sdk.AccAddress,
 	nonces []uint64,
 ) *MsgRegisterFee {
+	var withdrawAddr string
+	if withdrawal != nil {
```

Comment:
```suggestion
	if withdrawer != nil {
```

IMO there are too many variations `withdraw`, `withdrawal`, `withdrawer` for the same purpose, which I stumble over a bit while scanning through the code.